integration_test/test_packet_loss.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
You mean create another `TestController` Elixir app instead of the shell script? I guess that could be done, but I don't think it'd solve the issue you're referring to.

Some means of synchronisation (i.e. creation of a file) is necessary because we don't know how long does the browser action `:get_stats` take to complete. Before applying packet loss, there are supposed to be 15 such actions in 1 second intervals -- however, they usually take around 20-30 seconds to complete. Therefore, the test controller can't know in advance when it's supposed to apply the network effect, as it MUST be done after the first bunch of stat collecting and before the second one.

There are limited ways to accomplish this -- having one of the containers create a file in the right moment seemed like the easiest one. I can change the controller code to sync in another way (e.g. keep polling the containers for certain logs, and enable the loss whenever it finds them; or, possibly, another solution) if you believe it to be a better option...

Reply:
Well, technically, we could run `pumba netem` in one of the containers (the one to apply loss on, or maybe even the server), but that would require us to expose the Docker daemon socket to the container itself, which to me seems like an anti-pattern at best, and a major security risk at worst...

I've thought about the approach with nodes, perhaps it really is the best one of the bunch. There'd need to be some fidgeting around with exposing epmd ports and such, but I've no doubt it's doable. I'll try to implement that.

Reply:
First of all, we can move this to another PR.



I think, I would create a central point responsible for managing the whole test. This point would also be responsible for asking browsers for statistics :thinking: 

We can discuss this before implementing in more details

Reply:
Fine by me, we can tackle this problem later on.